src/main.js

```diff
@@ -175,28 +168,30 @@ function makePoster(){
 }
 
 function saveMainPoster() {
-  currentPoster = new Poster(posterImg.src, posterTitle.innerHTML, posterQuote.innerHTML);
-    for(var i = 0; i < savedPosters.length; i++ ) {
-      if (savedPosters[i].url == currentPoster.url &&
-        savedPosters[i].title == currentPoster.title &&
-        savedPosters[i].quote == currentPoster.quote) {
-        var existingPoster = savedPosters[i]
-      }
-    }
-    if (!existingPoster) {
-      savedPosters.push(currentPoster);
-      makePoster();
-    }
+  if (!existingPoster()) {
```

Comment:
Nice refactoring here!

Reply:
You're checking if existingPoster is truthy? or if it has a value? 
I don't think you need the parens, but not sure.

Reply:
We will try without! We thought because was a function we needed the parens.